web/src/components/filings-calendar/tax-access-modal/TaxAccessStepTwo.test.tsx

```diff
@@ -629,6 +598,38 @@ describe("<TaxAccessStepTwo />", () => {
         return expect(currentUserData().taskProgress["determine-naics-code"]).toEqual("COMPLETED");
       });
     });
+
+    it("updates municipality on successful response", async () => {
```

Comment:
[pebble] this reads as though `it` (the component) updates municipality... could maybe be renamed to say that it updates with municipality **from the API response** or something like that